src/main/java/de/hysky/skyblocker/mixins/PlayerListHudMixin.java

```diff
@@ -54,4 +55,9 @@ public class PlayerListHudMixin {
         }
     }
 
+    @Inject(at = @At("HEAD"), method = "setFooter")
+    public void skblocker$updateFooter(@Nullable Text footer, CallbackInfo info) {
+        PlayerListMgr.updateFooter(footer);
```

Comment:
Possibly remove the other updateFooter in the method above? No real need to have it twice, unless `setFooter` gets cancelled